assignments/#10.1.py

```diff
@@ -0,0 +1,17 @@
+
+#10.1 Split large number into hundreds, tens and units.
+
+
+print('This program split given large number into hundreds, tens and units.')

```

Comment:
corrected print function (should be backwards compatible between python2 and python3)